src/version.rc

```diff
@@ -23,12 +23,12 @@ BEGIN
         BEGIN
             VALUE "CompanyName", "\0"
             VALUE "FileDescription", "SDL_sound\0"
-            VALUE "FileVersion", "2, 0, 3, 0\0"
+            VALUE "FileVersion", "3, 0, 0, 0\0"
```

Comment:
`FILEVERSION` and `PRODUCTVERSION` above are wrong.

I'd suggest to port `build-scripts/test-versioning.sh` from the SDL satellite libraries to SDL_sound.

Reply:
Leaving this for later for now